tools/pontoon/search_filters.md

```diff
@@ -4,6 +4,8 @@
 
 ## Filters
 
+### String states
+
```

Comment:
Note that they are called "statuses", not "states" in the UI.

Reply:
Uhm, we need to update the rest of the page then. Fixing it.

Reply:
Actually, should we just go with `status` for plural? http://onlinewritingtraining.com.au/2017/04/plural-of-status/

Reply:
I'm ok with that.